src/Components/Buttons/ButtonVersionized.tsx

```diff
@@ -43,26 +43,26 @@ class ButtonVersionized extends React.Component<Props,IButtonVersionized>{
             disabledVersion: "",
             disabledCategory: "",
         }
-        this.updatePopUp = this.updatePopUp.bind(this);
+        this.changeShowPopUp = this.changeShowPopUp.bind(this);
     }
 
     /**
-     * updates the current showPopUp with the given value
-     * @param value
+     * Changes the current showPopUp with the given boolean value.
+     * @param boolean_value
      */
-    updatePopUp = (value) => {
-        this.setState({showPopUp: value})
+    changeShowPopUp = (boolean_value) => {
+        this.setState({showPopUp: boolean_value})
     }
 
     /**
-     * set the new version and button
+     * Handles version change when clicking on a button, i.e. sets the selected version, button and date or renders
+     * popup for non available versions.
      * @param version
-     * @param btn
      */
     handleVersionClick(version) {
         const DROPDOWN = document.getElementById(version);
```

Comment:
Do not use upper case for "const" as these are not proper constants in the traditional sense: https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/

Reply:
which is: not a compile time constant